assets/i18n/ar.json

```diff
@@ -689,6 +689,7 @@
 	"I18N_NO": "لا",
 	"I18N_ONE_SUBSCRIBER_TEXT": "لديك مشترك واحد.",
 	"I18N_PARTNERSHIPS_PAGE_BREADCRUMB": "شراكات",
+	"I18N_PARTNERSHIPS_PAGE_GET_STARTED_BUTTON_LINK": "https://docs-google-com.translate.goog/forms/d/e/1FAIpQLSdL5mjFO7RxDtg8yfXluEtciYj8WnAqTL9fZWnwPgOqXV-9lg/viewform?_x_tr_sl=en&_x_tr_tl=ar&_x_tr_hl=en-US&_x_tr_pto=wapp",
```

Comment:
This approach won't do, I think. Translators won't understand it and won't know how to translate it.

Instead, this key should not be included in the translation json files at all. Generating the correct link should be done within the main codebase itself.

Please fix.

Reply:
Also please check what happens if we use a language code that is not supported by Google Forms (like pcm for Naija, probably). Does it default to English?